setup.cfg

```diff
@@ -1,18 +1,16 @@
-[isort]
-skip =
-    drf-sources,
-    rest_framework-stubs,
-    test-data
-include_trailing_comma = true
-multi_line_output = 5
-wrap_length = 120
-
 [flake8]
-exclude =
-    drf-sources,
-    rest_framework-stubs,
-    test-data
+exclude = .*/
```

Comment:
Let's use explicit excludes. This is really hard to read / find.

Reply:
Ok